recipes-extended/xen/blktap3/compiler-errors-fix.patch

```diff
@@ -0,0 +1,409 @@
+################################################################################
+SHORT DESCRIPTION: 
+################################################################################
+Resolve compiler errors.
```

Comment:
This seems like multiple patches all merged together.  I think you should avoid the whitespace changes - that will cut down on the patch size.  Then maybe separate out the inline changes from the PRIu64 changes into there own patches.  Those should be easily upstreamable.  The remainder would then be pretty small.

Reply:
This patch should be upstream'd completely, if you suggest splitting them, I'll do so. Thanks